package.json

```diff
@@ -99,6 +97,9 @@
         "webpack-cli": "^5.1.4",
         "yjs": "^13.5.0"
     },
+    "resolutions": {
+        "parse5": "7.2.1"
```

Comment:
This is required because of some incompatibility if `parse5 >= 7.3` is used which is waiting a fix upstream.